src/status_im/contexts/chat/messenger/messages/view.cljs

```diff
@@ -13,13 +13,15 @@
 
 (defn- chat-screen
   [{:keys [insets] :as props}]
-  (let [alert-banners-top-margin (rf/sub [:alert-banners/top-margin])]
-    [rn/keyboard-avoiding-view
-     {:style                    style/keyboard-avoiding-container
-      :keyboard-vertical-offset (- (if platform/ios? alert-banners-top-margin 0) (:bottom insets))}
-     [list.view/messages-list-content props]
-     [messages.navigation/view props]
-     [composer.view/composer props]]))
+  (let [alert-banners-top-margin (rf/sub [:alert-banners/top-margin])
+        chat-exist?              (rf/sub [:chats/current-chat-exist?])]
+    (when chat-exist?
```

Comment:
empty screen will be opened?

Reply:
Thank you @flexsurfer for reviewing PR.
As you can see here https://github.com/status-im/status-mobile/blob/12515a49046938f688a75e89316e0d4149933dc5/src/status_im/contexts/chat/messenger/messages/view.cljs#L45, chat view is above placeholder/loading skeleton. So until chat is fetched user will just see loading skeleton.